### DISCUSSION TITLE
Some emails getting garbled

### DISCUSSION THREAD

**Discussion by brywhi** [postalserver/postal#1783] [Help with using Postal] [answered]
Not sure how to ask the right question on this one.

On a handful of systems we're using with Postal, we are seeing a strange behavior, where the emails come through all garbled with special characters, etc. I think this might be a charset issue maybe, but I have no clue. Here is an example of a failed plaintext email, sent using SMTP. Screenshot taken from the Postal UI. Any ideas where to start troubleshooting this?
![image](https://user-images.githubusercontent.com/42787119/147365066-b4bc9f1d-292d-4176-bd83-78c61b3acda1.png)

**Comment by willpower232** [comment]
If you're able to send some messages through your Postal server without issue then it implies that there is a problem with how you're sending the messages. Can you look in the database to see how widespread the issues are?

**Reply by brywhi** [reply]
It seems to be that the systems that are sending the garbled messages are using Base-64 encoding to transmit the message, and Postal isn't liking that. Is Base-64 supported by Postal?

**Comment by brywhi** [comment, marked as answer]
Looks like it is the same as this item:

https://github.com/postalserver/postal/discussions/1442

Disabling tracking fixed the issue, although this is not the ideal solution.

**Reply by willpower232** [reply]
Thanks for confirming, I appreciate its not ideal but I'm not sure how useful click tracking is when you don't send emails with the API.

**Reply by brywhi** [reply]
The bummer part is that we have to disable click tracking for all emails- right? Is there a way to only disable it for emails sent with SMTP?

We do use the click tracking to check on email deliverability/interaction rate for SMTP emails too by querying the SQL db- just to get broad metrics.

**Comment by Trapulo** [comment]
same problem here, sending from .NET Core Assembly System.Net.Mail, Version=6.0.0.0
